service-map/src/main/java/fi/nls/oskari/map/layer/formatters/LayerJSONFormatterUSERLAYER.java

```diff
@@ -54,4 +62,18 @@ private static String getUserLayerTileUrl() {
         }
         return PROPERTY_RENDERING_URL + "&id=";
     }
+
+    // creates JSONArray from JSONArray's JSONObject keys [{key,value},{key2,value2},..] -> [key1, key2,..]
+    private static JSONArray createJSONArrayJSONObjectKeys(final JSONArray json) {
```

Comment:
Normally yes but with this implementation probably not. This assumes that there's always exactly one key for each object in the array.